scripts/globals/pets/wyvern.lua

```diff
@@ -107,13 +109,19 @@ function onMobSpawn(mob)
                 for mod = 0, 5 do
                     if target:getMod(tpz.mod.FIREDEF + mod) < target:getMod(tpz.mod.FIREDEF + weakness) then
                         breaths = {}
-                        table.insert(breaths, 630 + mod)
+                        table.insert(breaths, tpz.jobAbility.FLAME_BREATH + mod)
                     elseif target:getMod(tpz.mod.FIREDEF + mod) == target:getMod(tpz.mod.FIREDEF + weakness) then
-                        table.insert(breaths, 630 + mod)
+                        table.insert(breaths, tpz.jobAbility.FLAME_BREATH + mod)
                     end
                 end
             else
-                breaths = {630, 631, 632, 633, 634, 635}
+                breaths = {tpz.jobAbility.FLAME_BREATH,
```

Comment:
per style guide, table braces on new line, spaces for indentation

```lua
breaths =
{
    tpz.jobAbility.FLAME_BREATH,
    tpz.jobAbility.FROST_BREATH,
    tpz.jobAbility.GUST_BREATH,
    tpz.jobAbility.SAND_BREATH,
    tpz.jobAbility.LIGHTNING_BREATH,
    tpz.jobAbility.HYDRO_BREATH,
}
```

Reply:
sounds good :+1:  thanks